packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/tvl-breakdown/_components/breakdown-page-wrapper.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+type Props = {
+  children: React.ReactNode
+}
+
+export function BreakdownPageWrapper({ children }: Props) {
+  return <div className="mx-auto h-full max-w-[928px]">{children}</div>
+}
```

Comment:
I guess it should be included in a layout

Reply:
https://github.com/l2beat/l2beat/pull/4855/commits/d34c14506d262288e7652587cddf0a172425eafb